sdk/resourcemanager/automanage/armautomanage/examples.go

```diff
@@ -0,0 +1,159 @@
+//go:build go1.18
+// +build go1.18
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is regenerated.
+// DO NOT EDIT.
```

Comment:
Please remove the comment for auto-generated code.

Reply:
Done